common/test/acceptance/tests/video/test_studio_video_module.py

```diff
@@ -42,6 +45,38 @@ def setUp(self):
         )
 
         self.assets = []
+        self.addCleanup(YouTubeStubConfig.reset)
+
+    def _create_course_unit(self, youtube_stub_config=None, subtitles=False):
+        """
+        Create a Studio Video Course Unit and Navigate to it.
+
+        Arguments:
+            youtube_stub_config (dict)
+            subtitles (bool)
+
+        """
+        if youtube_stub_config:
+            YouTubeStubConfig.configure(youtube_stub_config)
+
+        if subtitles:
+            self.assets.append('subs_3_yD_cEKoCk.srt.sjson')
+
+        self.navigate_to_course_unit()
+
+    def _create_video(self):
+        """
+        Create Xblock Video Component.
+        """
+        self.video.create_video()
+
+        video_xblocks = self.video.xblocks()
+
+        # Total video xblock components count should be equals to 2
+        # Why 2? One video component is created by default for each test. Please see
+        # test_studio_video_module.py:CMSVideoTest._create_course_unit
```

Comment:
Better to just reference the base class? The comment is very helpful btw thank you.